source/loader/layers/sanitizer/asan_statistics.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+ *
+ * Copyright (C) 2024 Intel Corporation
+ *
+ * Part of the Unified-Runtime Project, under the Apache License v2.0 with LLVM Exceptions.
+ * See LICENSE.TXT
+ * SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+ *
+ * @file asan_report.hpp
+ *
+ */
+
+#include "asan_statistics.hpp"
+#include "ur_sanitizer_layer.hpp"
+#include <atomic>
+
+namespace ur_sanitizer_layer {
+
+std::atomic<size_t> MallocedSize;
```

Comment:
Please don't use globals, they interfere with UR loader's ability to properly initialize and teardown.
Can't these variables be a part of the `AsanStats` struct?

Reply:
Sorry, these variables are unused, I'll remove them.

Reply:
thanks